contracts/strategies/bases/strategy-vtx-single-farm.sol

```diff
@@ -5,6 +5,8 @@ import "../strategy-joe-base.sol";
 import "../../interfaces/vtx.sol";
 import "../../interfaces/wavax.sol";
 
+import "hardhat/console.sol";
```

Comment:
Do you want to leave `console.log`s in the code? I had a contract fail to compile the ABI because of this, I would remove unless it's needed